tools/benchmarking/utils/metrics.py

```diff
@@ -0,0 +1,193 @@
+"""Methods to compute and save metrics."""
+
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+import random
+import string
+import time
+from glob import glob
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Dict, List, Union
+
+import pandas as pd
+from omegaconf import DictConfig, ListConfig
+from pytorch_lightning import Trainer
+from torch.utils.tensorboard.writer import SummaryWriter
+
+import wandb
+from anomalib.data import get_datamodule
+from anomalib.models import get_model
+from anomalib.utils.sweep import (
+    get_meta_data,
+    get_openvino_throughput,
+    get_sweep_callbacks,
+    get_torch_throughput,
+)
+
+from .convert import convert_to_openvino
+
+
+def get_single_model_metrics(model_config: Union[DictConfig, ListConfig], openvino_metrics: bool = False) -> Dict:
```

Comment:
Maybe we could move this to `benchmarking.py`, since it is the place where most of the execution time of the benchmarking will be spent. So maybe it's a bit too fundamental to have it in utils. What do you think?